drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -146,4 +149,54 @@ def allowed_pick_list_references
       enums.to_set
     end
   end
+
+  def check_cdeds(document, role)
+    owner_type = role ? Hmis::Form::Definition.owner_class_for_role(role)&.sti_name : nil
```

Comment:
if owner_type is `nil`, we can't really check anything.  We probably shouldn't call this function if we don't have a role.